modules/distribution/src/repository/conf/data-bridge/data-bridge-config.xml

```diff
@@ -1,72 +1,74 @@
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
   -->
 
 <dataBridgeConfiguration>
 
-	<workerThreads>10</workerThreads>
-	<eventBufferCapacity>10000</eventBufferCapacity>
-	<clientTimeoutMin>30</clientTimeoutMin>
+    <StreamDefinitionStore>
+        org.wso2.carbon.databridge.streamdefn.cassandra.datastore.CassandraStreamDefinitionStore
+    </StreamDefinitionStore>
 
-	<dataReceiver name="Thrift">
-		<config name="tcpPort">7611</config>
-		<config name="sslPort">7711</config>
-	</dataReceiver>
+    <workerThreads>10</workerThreads>
+    <eventBufferCapacity>10000</eventBufferCapacity>
+    <clientTimeoutMS>30000</clientTimeoutMS>
+    <keySpaceName>EVENT_KS</keySpaceName>
 
-	<dataReceiver name="Binary">
-		<config name="tcpPort">9611</config>
-		<config name="sslPort">9711</config>
-		<config name="sslReceiverThreadPoolSize">100</config>
-		<config name="tcpReceiverThreadPoolSize">100</config>
-	</dataReceiver>
+    <!-- Default configuration for thriftDataReceiver -->
+    <thriftDataReceiver>
+        <!--<hostName>localhost</hostName>-->
+        <port>7611</port>
+        <securePort>7711</securePort>
+    </thriftDataReceiver>
 
-	<!--<streamDefinitions>
-		<streamDefinition>
-			{
-			 'name':'org.wso2.esb.MediatorStatistics',
-			 'version':'1.3.0',
-			 'nickName': 'Stock Quote Information',
-			 'description': 'Some Desc',
-			 'metaData':[
-			 {'name':'ipAdd','type':'STRING'}
-			 ],
-			 'payloadData':[
-			 {'name':'symbol','type':'STRING'},
-			 {'name':'price','type':'DOUBLE'},
-			 {'name':'volume','type':'INT'},
-			 {'name':'max','type':'DOUBLE'},
-			 {'name':'min','type':'Double'}
-			 ]
-			}
-		</streamDefinition>
-		<streamDefinition domainName="wso2">
-			{
-			 'name':'org.wso2.esb.MediatorStatistics',
-			 'version':'1.3.4',
-			 'nickName': 'Stock Quote Information',
-			 'description': 'Some Other Desc',
-			 'metaData':[
-			 {'name':'ipAdd','type':'STRING'}
-			 ],
-			 'payloadData':[
-			 {'name':'symbol','type':'STRING'},
-			 {'name':'price','type':'DOUBLE'},
-			 {'name':'volume','type':'INT'}
-			 ]
-			}
-		</streamDefinition>
-	</streamDefinitions>-->
+    <!--<streamDefinitions>
+        <streamDefinition>
+            {
+             'name':'org.wso2.esb.MediatorStatistics',
+             'version':'1.3.0',
+             'nickName': 'Stock Quote Information',
+             'description': 'Some Desc',
+             'metaData':[
+             {'name':'ipAdd','type':'STRING'}
+             ],
+             'payloadData':[
+             {'name':'symbol','type':'STRING'},
+             {'name':'price','type':'DOUBLE'},
+             {'name':'volume','type':'INT'},
+             {'name':'max','type':'DOUBLE'},
+             {'name':'min','type':'Double'}
+             ]
+            }
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.